docs/features/multidomain.rst

```diff
@@ -0,0 +1,333 @@
+   and/or the config mode is skipped, so they do not make sense in a
+   domain specific way:
+
+   -  authorized_keys
+   -  poe_passthrough
+   -  mesh_on_wan
+   -  mesh_on_lan
+   -  single_as_lan
+   -  setup_mode.skip
+   -  autoupdater.branch
+   -  mesh_vpn.enabled
+   -  mesh_vpn.pubkey_privacy
+   -  mesh_vpn.bandwidth_limit
+   -  mesh_vpn.bandwidth_limit.enabled
+   -  mesh_vpn.bandwidth_limit.ingress
+   -  mesh_vpn.bandwidth_limit.egress
+
+-  Variables that influence the appearance of the config mode,
+   domain-independent because they are relevant before a domain was selected.
+
+   -  config_mode.owner.obligatory
+   -  config_mode.geo_location.show_altitude
+   -  config_mode.remote_login
+   -  config_mode.remote_login.show_password_form
+   -  config_mode.remote_login.min_password_length
+   -  hostname_prefix
+   -  mesh_vpn.fastd.configurable
+   -  roles.default
+   -  roles.list
+
+-  Specific to a firmware build itself:
+
+   -  site_name
+   -  autoupdater.branches.*.name
+   -  autoupdater.branches.*.good_signatures
+   -  autoupdater.branches.*.pubkeys
+
+-  We simply do not see any reason, why these variables could be helpful
+   in a domain specific way:
+
+   -  mesh_vpn.fastd.syslog_level
+   -  wifi*.ibss.supported_basic_rates
+   -  wifi*.mesh.supported_basic_rates
+   -  timezone
+   -  regdom
+
+domain.conf only variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+-  Obviously:
+
+   -  domain_names
+
+      -  a table of domain codes to domain names
+         ``domain_names = { foo = 'Foo Domain', bar = 'Bar Domain', baz = 'Baz Domain' }``
+
+   -  hide_domain
+
+      -  prevents a domain name(s) from appearing in config mode, either
+         boolean or array of domain codes
+
+         -  ``true``, ``false``
+         -  ``{ 'foo', 'bar' }``
+
+-  Because each domain is considered as an own layer 2 network, these
+   values should be different in each domain:
+
+   -  next_node.mac
+   -  next_node.ip4
+   -  next_node.ip6
+   -  next_node.name
+   -  prefix6
+   -  prefix4
+   -  extra_prefixes6
+
+-  To prevent accidential bridging of different domains, all meshing
+   technologies should be seperated:
+
+   -  domain_seed (wired mesh)
+
+      -  must be a random value used to derive the vxlan id for wired meshing
+
+   -  wifi*.ibss.ssid
+   -  wifi*.ibss.bssid
+   -  wifi*.mesh.id
+   -  mesh_vpn.fastd.groups.*.peers.remotes
+   -  mesh_vpn.fastd.groups.*.peers.key
+   -  mesh_vpn.tunneldigger.brokers
+
+-  Clients consider WiFi networks sharing the same ESSID as if they were
+   the same L2 network and try to reconfirm and reuse previous
+   addressing. If multiple neighbouring domains shared the same ESSID,
+   the roaming experience of clients would degrade.
+
+   -  wifi*.ap.ssid
+
+Example config
+--------------
+
+site.conf
+^^^^^^^^^
+
+.. code:: lua
+
+    {
+        site_name = 'Centauri Mesh',
+        site_code = 'centauri',
+        default_domain = 'alpha_centuari',
```

Comment:
"alpha_centuari", should be "alpha_centauri"

Reply:
This is still unfixed.